### ISSUE TITLE
Java exception in new version

### ISSUE THREAD

**Issue by jongbelegen** [reasonml-editor/reasonml-idea-plugin#180]
plugin version: v0.78.4

### Description

On the new version i get the following error after changing some code and saving it:
```
java.lang.IllegalArgumentException: Argument for @NotNull parameter 'line' of com/reason/build/bs/compiler/BsProcess.ocamlVersionExtractor must not be null
	at com.reason.build.bs.compiler.BsProcess.$$$reportNull$$$0(BsProcess.java)
	at com.reason.build.bs.compiler.BsProcess.ocamlVersionExtractor(BsProcess.java)
	at com.reason.build.bs.compiler.BsProcess.getOCamlVersion(BsProcess.java:189)
	at com.reason.hints.InsightManagerImpl.getRincewindFilename(InsightManagerImpl.java:76)
	at com.reason.hints.InsightManagerImpl.getRincewindFile(InsightManagerImpl.java:60)
	at com.reason.hints.InsightManagerImpl.queryTypes(InsightManagerImpl.java:83)
	at com.reason.ide.hints.InferredTypesService.queryForSelectedTextEditor(InferredTypesService.java:65)
	at com.reason.build.bs.compiler.BsOutputListener.lambda$processTerminated$0(BsOutputListener.java:79)
	at com.intellij.openapi.application.TransactionGuardImpl$2.run(TransactionGuardImpl.java:315)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.doRun(LaterInvocator.java:435)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.runNextEvent(LaterInvocator.java:419)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.run(LaterInvocator.java:403)
	at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:311)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:764)
	at java.awt.EventQueue.access$500(EventQueue.java:98)
	at java.awt.EventQueue$3.run(EventQueue.java:715)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:74)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:734)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:757)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:706)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:375)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:205)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
System:

  Model Name:	MacBook Pro
  Model Identifier:	MacBookPro14,1
  Processor Name:	Intel Core i5
  Processor Speed:	2,3 GHz
  Number of Processors:	1
  Total Number of Cores:	2
  L2 Cache (per Core):	256 KB
  L3 Cache:	4 MB
  Hyper-Threading Technology:	Enabled
  Memory:	8 GB

  System Version:	macOS 10.14.5 (18F132)
  Kernel Version:	Darwin 18.6.0
```

**Comment by jongbelegen** [comment]
Fix: After updating intellij to the latest version this plugin works again.

**Comment by giraud** [comment]
fixed in 0.79
